RetroRabbit/RGO-Client
# Sprint 1 Completion

- Added Angular Project
- login implementation
- Added routing from sign in to home page
- login testing
- testing
- login with google works. no state management
- Apply suggestions from code review
- Merged PR 4809: angular login with google
- called api to check user exists
- Merge branch 'Frontend-DEV' into feature/connect-to-api
- added response type and moved the piping to the service
- state management initial setup
- testing state
- Added state management
- Merge branch 'Frontend-DEV' into feature/stateManagement
- Merged PR 4813: Login
- Merge branch 'Frontend-DEV' into feature/stateManagement
- Added material dashboard
- Merged PR 4833: Implementing state manangement
- Merged PR 4835: Added material dashboard
- login page styled with material dashboard
- grad to do interface
- Revert "grad to do interface"
- Revert "Revert "grad to do interface""
- Merge branch 'Frontend-DEV' into Setup/design
- Adding Navbar
- Merged PR 4854: grad to do interface
- Update sign-in.component.html
- Merged PR 4860: login page styled with material dashboard
- Adding Logout Button
- Code cleanup, refactor
- Merge remote-tracking branch 'origin/Frontend-DEV' into Navbar
- Merge remote-tracking branch 'origin/Refactor/DevRefactor-Matthew' into Navbar
- Merged PR 4868: Code Cleanup
- Apply suggestions from code review
- Apply suggestions from code review
- Merged PR 4869: Adding Navbar
- Adding actoin for getting the type of user
- Changing Serveice to return user type
- Passing user type to Token
- Adding reference to the Token
- converting string to number
- adding type to Token
- Defining GetType
- Adding Conditional Redering
- removing unsed images
- fixing error related to material dashboard
- Apply suggestions from code review
- Removing GetType
- simplyfing GetUserType
- Merge branch 'feature/Super-User-Sign-In' of https://dev.azure.com/retro-rabbit/RetroGradOnboard/_git/RGO-Client into feature/Super-User-Sign-In
- Merged PR 4884: Adding Sign in dashboard
- Added side bar and header
- resolving comments
- resolving comments
- Merged PR 4905: Side bar
- Added Workshop Compomnent
- Added comp to Sidebar
- Added routing to home comp
- Added routing to sidebar
- removed commented code
- Removed unused routing path
- Merged PR 4913: Added routing a comp to home comp from sidebar
- Added Personal Project page
- Merge branch 'Dev' into dvanhout/feature/personal-project-page
- created events page
- Added mentor personal project sidebar item
- Updated sidebar component
- removed comments
- Added personlproject component
- Updated Personal Project Page
- Merged PR 4921: Added mentor personal project sidebar item
- Merge branch 'Dev' into dvanhout/feature/personal-project-page
- Merge branch 'Dev' into feature/EventsPage
- bug fixes
- Update sidebar.component.ts
- Merged PR 4927: Adding events to sidebar
- Adding Admin Signin permissions
- Merged PR 4925: Updated Personal Project Page
- Merge branch 'Dev' into feature/Admin-Signin
- Removing duplicate import
- Adding icons
- Merged PR 4937: Adding Admin Signin
- [AzureDevOps/#5371](https://dev.azure.com/retro-rabbit/RetroGradOnboard/_workitems/edit/5371) Sidebar navigate to forms on Grad
- [AzureDevOps/#5339](https://dev.azure.com/retro-rabbit/RetroGradOnboard/_workitems/edit/5339) - Create Workshop Component
- Merged PR 4941: Side Navigation forms on Grad
- Merge branch 'Dev' into feature/WorkshopComponent-MatthewSchoeman
- [AzureDevOps/#5339](https://dev.azure.com/retro-rabbit/RetroGradOnboard/_workitems/edit/5339) - Create Workshop Component
- Merged PR 4954: Added workshop component

-----
_Azure DevOps work item(s) linked to the original pull request:_
- [AzureDevOps/#5133](https://dev.azure.com/retro-rabbit/_workitems/edit/5133)
- [AzureDevOps/#5339](https://dev.azure.com/retro-rabbit/_workitems/edit/5339)
- [AzureDevOps/#5371](https://dev.azure.com/retro-rabbit/_workitems/edit/5371)
